### PR TITLE
Zoey_Completed html-forms

### PR DESCRIPTION
**Pull request by Zoey1318**
html-forms practice

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        <input type="radio" name="language" value="javascript">
+        JavaScript
+      </label>
+
+      <p>Would you like to sign up for more information?</p>
+      <label>
+        <input type="checkbox" name="sign up" value="sign up"> Sign me up!
+      </label><br/ >
+
+      <label for="comments">
```
Comment (scrumptiousdev) [inline html-forms/index.html:47]: When we don't wrap the input control with `label` we need to connect those two with `for` and which one??

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Good job overall! One small change!

File: html-forms/index.html (hunk 1 above)
Comment (Zoey1318) [reply html-forms/index.html:47]: input id? so for line 49 since there is no input do i need use textarea id="comments"?

Comment (scrumptiousdev) [reply html-forms/index.html:47]: Yes indeed!

**Comment by Zoey1318** [comment]
recommit_html-forms

**Review by scrumptiousdev — APPROVED** [review]
🪄 Approved!

**Pull request merged**